### PR TITLE
test-case: add kernel log check after module insertion

### PR DESCRIPTION
**Pull request by aiChaoSONG**
Previously, we didn't check dmesg log after module insertion, so test case may passed even there is error in dmesg after module insertion. This patch fixes that condition.

### CODE REVIEW

**Comment by aiChaoSONG** [comment]
@xiulipan Ping for merge

File: test-case/check-kmod-load-unload.sh [hunk 1]
```diff
@@ -89,6 +89,9 @@ do
     ## - 2a: check for errors after insertion
     dlogi "checking for general errors after kmod insert with sof-kernel-log-check tool"
     sof-kernel-log-check.sh $KERNEL_LAST_LINE
+    [[ $? -ne 0 ]] && \
+        dloge "Found error(s) in kernel log after module insertion" && \
```
Comment (xiulipan) [inline test-case/check-kmod-load-unload.sh]: no need to new line here. This look worse then a long line.

**Comment by libinyang** [comment]
LGTM

File: test-case/check-kmod-load-unload.sh (hunk 1 above)
Comment (aiChaoSONG) [reply test-case/check-kmod-load-unload.sh]: replace with if else, and align code style of the whole file

**Pull request merged**